brainstormforce/ultimate-addons-for-gutenberg
# Fix: Admin Dashboard - Resolved console issue for `blockCategories` undefined.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/2634da90-a8d4-4a2c-9d29-a36878a5130f?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- See screencast - https://share.getcloudapp.com/jkuvmzY7

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Go to Spectra.
- Click on blocks Tabs and check the console. Is the blocks section is visible or not.

### Checklist:
